go.mod

```diff
@@ -14,10 +14,9 @@ require (
 	github.com/huandu/xstrings v1.3.2 // indirect
 	github.com/imdario/mergo v0.3.12
 	github.com/jenkins-x-plugins/jx-charter v0.0.28
-	github.com/jenkins-x/gen-crd-api-reference-docs v0.1.6 // indirect
-	github.com/jenkins-x/go-scm v1.10.10
+	github.com/jenkins-x/go-scm v1.10.11
 	github.com/jenkins-x/jx-api/v4 v4.3.0
-	github.com/jenkins-x/jx-helpers/v3 v3.1.0
+	github.com/jenkins-x/jx-helpers/v3 v3.1.1
 	github.com/jenkins-x/jx-kube-client/v3 v3.0.2
 	github.com/jenkins-x/jx-logging/v3 v3.0.6
 	github.com/jenkins-x/lighthouse-client v0.0.233
```

Comment:
Still needs the lighthouse release pipeline to succeed before I can bump this ...

Reply:
I think that can be skipped for now, jx-helpers and go-scm were more important.